tutorials/commands/ssh-across-clusters.rst

```diff
@@ -0,0 +1,156 @@
+
+We now have defined a goal for success - getting this listing working by running a command 
+from a different cluster node.
+
+-----------------------
+Connect to the Instance
+-----------------------
+
+Next, let's ssh into another cluster.  Take the hostname where your instance is running,
+and create a `proxy jump <https://en.wikibooks.org/wiki/OpenSSH/Cookbook/Proxies_and_Jump_Hosts>`_ in your ``~/.ssh/config``:
+
+.. code-block:: ssh 
+
+    Host noodle
+        HostName noodle
+
+    Host noodle220
+        hostname noodle220
+        ProxyJump noodle
+
+.. note::
+
+  This ``~/.ssh/config`` needs to be written on the cluster system where you are going to connect from.
+  In many cases, the shared filesystem could map your home across clusters so you can see the file in
+  multiple places.
+
+
+You'll first need to tell Flux to use ssh for the proxy command:
+
+.. code-block:: ssh 
+
+    quartz:~$ export FLUX_SSH=ssh
+
+Next, from this same location, try using ``flux proxy`` to connect to your Flux Instance! Target the URI
+that you found before, ``local:///var/tmp/flux-MLmxy2/local-0``, and add the hostname ``noodle220`` to the address:
+
+.. code-block:: sh
+
+     quartz:~$ flux proxy ssh://noodle220/var/tmp/flux-MLmxy2/local-0
+
+If you have trouble - use the force!
+
+.. code-block:: sh
+
+     quartz:~$ flux proxy --force ssh://noodle220/var/tmp/flux-MLmxy2/local-0
+
+
+You should then be able to run the same resource list:
+
+.. code-block:: sh
+
+    quartz:~$ flux resource list
+         STATE NNODES   NCORES    NGPUS NODELIST
+          free      4      160        0 noodle[220,220,220,220]
+     allocated      0        0        0 
+          down      0        0        0 
+
+Next, try submitting a job! You should be able to see that you are running on the first cluster,
+but from the second. 
+
+.. code-block:: sh
+
+    quartz:~$ flux mini run hostname
+    noodle220
+
+If you are still connected to the first, you should also be able to query the jobs.
+E.g., here we submit a sleep from the second connected cluster:
+
+.. code-block:: sh
+
+    quartz:~$ flux mini submit sleep 60
+    f22hdyb35
+
+And then see it from either cluster node!
+
+.. code-block:: sh
+
+    $ flux job list  | jq
+    {
+      "id": 2272725565440,
+      "userid": 34633,
+      "urgency": 16,
+      "priority": 16,
+      "t_submit": 1675713045.009863,
+      "state": 16,
+      "name": "sleep",
+      "ntasks": 1,
+      "nnodes": 1,
+      "ranks": "2",
+      "nodelist": "noodle220",
+      "expiration": 1676317845,
+      "t_depend": 1675713045.009863,
+      "t_run": 1675713045.0290241,
+      "annotations": {
+        "sched": {
+          "queue": "default"
+        }
+      }
+    }
+
+And that's it! With this strategy, it should be easy to interact with Flux instances from
+two resources where ssh is supported. If you have any questions, please `let us know <https://github.com/flux-framework/flux-docs/issues>`_.
```

Comment:
> If you have any questions, please `let us know <https://github.com/flux-framework/flux-docs/issues>`_.

Instead of having something like this in every tutorial, perhaps we just need a header/footer comment or something within the "tutorials" page?

Reply:
or ... if people get to this page via search engines, perhaps it good to have at the bottom.

Reply:
But then if they are in a specific tutorial, they wouldn't see it right? It's important (I think) to be at the bottom of the page so the reader might glance through the tutorial, and then feel like they are missing something / have a question and immediately see that.